components/event-bus/cmd/event-bus-publish-knative/handlers/publish.go

```diff
@@ -150,17 +156,29 @@ func generateEventID() (string, error) {
 	return uid.String(), err
 }
 
-func buildCloudEvent(publishRequest *api.PublishRequest, traceContext *api.TraceContext) *api.CloudEvent {
-	cloudEvent := &api.CloudEvent{}
-	cloudEvent.PublishRequest = *publishRequest
+func buildCloudEvent(publishRequest *api.PublishRequest, traceContext *api.TraceContext) *Message {
```

Comment:
Does it make sense to rename **buildCloudEvent** to **buildMessage**?

Reply:
Done